nucliadb_node/src/settings.rs

```diff
@@ -86,16 +91,44 @@ const DEFAULT_ENV: &str = "stage";
 #[derive(Clone)]
 pub struct Settings {
     env: Arc<EnvSettings>,
+    pub object_store: Arc<dyn ObjectStore>,
```

Comment:
I think adding `object_store` into `Settings` pollutes the struct. Maybe a `Context` or `GlobalContext` containing both settings and object storage would be cleaner?

Reply:
Agree! In fact, we looked into it with Javi but the changes went far out of the scope of the task at hand.
Feel free to ship a proposal for this!